Content.Server/Voting/Managers/VoteManager.cs

```diff
@@ -270,7 +271,7 @@ private void SendSingleUpdate(VoteReg v, ICommonSession player)
             for (var i = 0; i < msg.Options.Length; i++)
             {
                 ref var entry = ref v.Entries[i];
-                msg.Options[i] = ((ushort) entry.Votes, entry.Text);
+                msg.Options[i] = (v.ShowVotes ? (ushort) entry.Votes : (ushort)0, entry.Text);
```

Comment:
```suggestion
                msg.Options[i] = (v.ShowVotes ? (ushort) entry.Votes : (ushort) 0, entry.Text);
```

Reply:
From what I see ss14 generally prefers no space between the cast, but you are right that this should conform to this should conform with the rest of the file.